.github/policies/close-stale-issues.yml

```diff
@@ -0,0 +1,55 @@
+id: close-stale-issues
+name: Label and Close Stale Issues
+description: Label 'stale' issues and close them after another period of inactivity
+resource: repository
+configuration:
+  resourceManagementConfiguration:
+    scheduledSearches:
+      - description: Apply 'stale' label to issues not updated for 6 years
+        frequencies:
+          - weekday:
+              day: Monday
+              time: 12:00
+        filters:
+          - isIssue
+          - isOpen
+          - noActivitySince:
+              days: 2192
+          - isNotLabeledWith:
+              label: stale
+        actions:
+          - addReply:
+              reply: >-
+                Due to lack of recent activity, this issue has been labeled as 'stale'. It will be closed if no further activity occurs within 30 more days. Any new comment will remove the label.
+          - addLabel:
+              label: stale
+      - description: Close 'stale' issues that have not been updated for 30 days
+        frequencies:
+          - weekday:
+              day: Wednesday
+              time: 16:00
```

Comment:
any reason you picked Monday 12 and Wednesday 16 for these?

Reply:
Monday so that the comment is laid down at the beginning of the week. Wednesday so that it would be 30 days (and one hour) after the comment is laid down and the label is applied. This puts both events in the middle of the work week.